src/org/thoughtcrime/securesms/util/ViewUtil.java

```diff
@@ -137,6 +137,9 @@ public static void fadeIn(final @NonNull View view, final int duration) {
     final SettableFuture future = new SettableFuture();
     if (view.getVisibility() == visibility) {
       future.set(true);
+    } else if (AccessibilityUtil.areAnimationsDisabled(view.getContext())) {
+      view.setVisibility(visibility);
+      future.set(true);
```

Comment:
this fixes a bug that the animation of the send-button wasn't disabled with system animations are disabled.